service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsMidEventControllerTest.java

```diff
@@ -97,24 +110,25 @@ void shouldCalculateFeeForSelectedOrderBundlesAndAddAmountToPayField(boolean ser
 
         verify(feeService).getFeesDataForAdditionalApplications(feeTypes);
         assertThat(response.getData())
-            .containsKeys("temporaryC2Document", "othersSelector")
+            .containsKeys("temporaryC2Document", "personSelector")
             .containsEntry("amountToPay", "1000")
             .containsEntry("displayAmountToPay", YES.getValue());
 
         if (servingOthersToggledOn) {
-            assertThat(String.valueOf(response.getData().get("hasOthers"))).isEqualTo("Yes");
-            assertThat(String.valueOf(response.getData().get("others_label")))
-                .contains("Other 1: test1", "Other 2: test2");
-            assertThat(extractCaseData(response).getOthersSelector()).isEqualTo(Selector.newSelector(2));
+            assertThat(String.valueOf(response.getData().get("hasRespondentsOrOthers"))).isEqualTo("Yes");
+            assertThat(String.valueOf(response.getData().get("people_label"))).contains(
+                "Person 1: Respondent 1 - John Smith\nPerson 2: Other 1 - test1\nPerson 3: Other 2 - test2\n");
+            assertThat(extractCaseData(response).getPersonSelector()).isEqualTo(Selector.newSelector(3));
         } else {
             assertThat(response.getData())
-                .doesNotContainKeys("hasOthers", "others_label")
-                .containsEntry("othersSelector", null);
+                .doesNotContainKeys("hasRespondentsOrOthers", "people_label")
+                .containsEntry("personSelector", null);
```

Comment:
couldn't you check for non-existence here, instead of checking for a null value?

Reply:
I couldn't do it as the request is created with CaseData and it has the `personSelector` as null. I can build the data with Map object, but I'll hard code the keys and will not be easy to read. so, I prefer to keep it like this. is it ok?

Reply:
Oh, I see. Just keep it. It's fine.